packages/calling/src/CallingClient/calling/types.ts

```diff
@@ -155,6 +155,11 @@ export enum TransferType {
   CONSULT = 'CONSULT',
 }
 
+export enum MUTE_TYPE {
```

Comment:
Could we use the new way of doing enums?

Reply:
Changing it for just this and having other enums in the old format is alright. For new SDKs we can follow the appaorach but for calling SDK also if we do it then it will be 2 different formats.